Scripts/symbolic_hermitians/generate_code.py

```diff
@@ -159,7 +159,7 @@ def delete_generated_files():
     deposit_vars = ["N","Fx","Fy","Fz"]
```

Comment:
This essentially reverts the EOMs to what they were before the whole L business. Now L is no longer used in the EOMs but rather is just recorded as a constraint for the particle evolution.